doctrine/doctrine-migrations-bundle/1.2/config/packages/doctrine_migrations.yaml

```diff
@@ -1,2 +1,5 @@
 doctrine_migrations:
     dir_name: '%kernel.project_dir%/src/Migrations'
+    # namespace is arbitrary but should be different from App\Migrations
+    # as migrations classes should NOT be autoloaded
+    namespace: DoctrineMigrations
```

Comment:
Just asking ... do we really need this `namespace` option? The value is "fake" and the app doesn't need it. Would it be possible to upgrade  the bundle to remove this option or accept null values?

Reply:
We don't want to risk people accidentally writing un-namespaced code...

Reply:
The namespace is required by Doctrine. Proper fix should probably be to "fix" the upstream bundle/library to respect PSR-4, but as migrations are used by a lot of people, I think having a temporary solution in Flex is good. And again, whenever this is fixed upstream, it will just be a matter of changing this default.

Reply:
@teohhanhui Having no namespace is not option anyway, that does not work with Doctrine.